util-logging/src/main/scala/com/twitter/logging/FileHandler.scala

```diff
@@ -224,9 +224,10 @@ class FileHandler(
       }
       case Policy.Weekly(weekday) => {
         next.set(Calendar.HOUR_OF_DAY, 0)
-        do {
+        while({ // rewritten from do-while-loop
```

Comment:
not a fan of scala3 rewrite of this. Can we do it ourselves?

```
next.add(Calendar.DAY_OF_MONTH, 1)
while (next.get(Calendar.DAY_OF_WEEK) != weekday) {
   next.add(Calendar.DAY_OF_MONTH, 1)
}
```

Reply:
Yeah, I didn't like it either but didn't spend more time thinking about it. I'll change it to your suggestion.